docs/deployment/rds-s3/README.md

```diff
@@ -286,7 +286,81 @@ mysql> select * from observation_logs;
+
+### 5.1 Setup the secrets access
+1. Navigate inside the `secrets` directory.
+```
+cd secrets
+```
+2. Install the script dependencies.
+```shell
+pip install -r requirements.txt
+```
+3. Execute the script, replace `YOUR_CLUSTER_REGION`, `YOUR_CLUSTER_NAME` and `YOUR_NAMESPACE` with the appropriate values.  
+Note that `YOUR_NAMESPACE` represents the namespace the secrets will be setup for.  
+For instance if your notebooks and pipelines will be in `kubeflow-user-example-com`, then you would use `kubeflow-user-example-com` in place of `YOUR_NAMESPACE`.
+```shell
+python secrets-setup.py --region YOUR_CLUSTER_REGION --cluster YOUR_CLUSTER_NAME --namespace YOUR_NAMESPACE
+```  
+You can always do `python secrets-setup.py --help` to learn more about the parameters available.    
+
+**Note :** The namespace must exist before executing the script.   
+
+### 5.2 (Optional) Set PodDefault selected by default in notebook UI  
+The Kubeflow notebook UI allows to select credentials configurations when creating a notebook, by default no configuration is selected but we can add the `PodDefault` label created in the previous step to the list of selected configuration.  
+This results in having the `PodDefault` already selected when creating a new notebook, otherwise the user must select it manually in the notebook UI.  
+  
+This change can be done before or after installing Kubeflow, here is how to make this modification depending on these 2 options.  
+
+#### 5.2.1 **Option 1: Update notebook UI config before installing Kubeflow**
+Modify the file `awsconfigs/apps/jupyter-web-app/configs/spawner_ui_config.yaml`
+```yaml
+  configurations:
+    # List of labels to be selected, these are the labels from PodDefaults
+    value:
+      - add-aws-secret
+```  
+#### 5.2.2 **Option 2: Update notebook UI config after installing Kubeflow**  
+We can update the notebook UI config at runtime by using the following command :  
+```shel
+kubectl edit $(kubectl get cm -n kubeflow -l app=jupyter-web-app -o=name | grep 'web-app-config') -n kubeflow
+```  
+
+Modify the configuration :  
+```yaml
+  configurations:
+    # List of labels to be selected, these are the labels from PodDefaults
+    value:
+      - add-aws-secret
+```  
+  
+Once you are done making the changes you can save and exit your editor and restart the notebook deployment to apply the changes.   
+
+```shell
+kubectl rollout restart deployment jupyter-web-app-deployment -n kubeflow
+```
+### 5.3 Verify notebook credentials configuration  
+To verify that your `PodDefault` setup was done successfully, navigate to the notebook creation UI and make sure you can select the `PodDefault`.  
+![](https://user-images.githubusercontent.com/26939775/155630906-0eecf1d9-3fb1-4d01-a85e-1cff46dc37e9.png)  
+Then create a notebook and check that the environment variables are accessible :  
+```python
+import os
+
+print(os.environ['AWS_ACCESS_KEY_ID'])
```

Comment:
nit: database port instead of sensitive information

Reply:
You're right, done.